example-new-architecture/src/App.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { type FC } from 'react';
+import { StatusBar } from 'react-native';
+import {
+  createStackNavigator,
+  type StackNavigationOptions,
+} from '@react-navigation/stack';
+import { NavigationContainer } from '@react-navigation/native';
+import { NavigationRoute, type RootStackParamList } from './routes';
+import DojoSdk from './screens/DojoSdk';
+import Settings from './screens/Settings';
+import SettingsProvider from './settings/SettingsProvider';
+
+const StackNavigator = createStackNavigator<RootStackParamList>();
+
+const dojoSdkScreenOptions: StackNavigationOptions = {
+  title: 'Dojo Sdk',
+};
+
+const settingsScreenOptions: StackNavigationOptions = {
+  title: 'Settings',
+};
+
+const App: FC = () => {
```

Comment:
Would be nice to go with a monorepo to share this stuff across example apps - guess when the old one is removed we wont need to worry.

Reply:
It would, something to think about down the line if old arch proves to stick around for a while - we should look into some branding in the example as well at some point